ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/QuickFixes/AddSetterFixTest.fs

```diff
@@ -11,6 +13,11 @@ type AddSetterFixTest() =
     [<Test>] member x.``Auto property 02 - Static``() = x.DoNamedTest()
     [<Test>] member x.``Auto property 03 - With getter``() = x.DoNamedTest()
     [<Test>] member x.``Auto property 04 - Abstract``() = x.DoNamedTest()
+    [<Test>] member x.``Auto property 05 - With getter - Access modifier``() = x.DoNamedTest()
+
+    // TODO: fix in the formatter
```

Comment:
Thanks for adding this todo here! I was going to suggest it while looking at the test data :)